op-node/rollup/finality/finalizer.go

```diff
@@ -34,9 +34,26 @@ const defaultFinalityLookback = 4*32 + 1
 // We do not want to do this too often, since it requires fetching a L1 block by number, so no cache data.
 const finalityDelay = 64
 
+// FinalizerConfig contains runtime configuration for the finalizer.
+type FinalizerConfig struct {
```

Comment:
Let's remove the redundant `Finalizer`, the package name already prefixes the struct in other package where it's imported.

```suggestion
// Config contains runtime configuration for the finalizer.
type Config struct {
```